q2_feature_classifier/_vsearch.py

```diff
@@ -148,14 +165,33 @@ def classify_hybrid_vsearch_sklearn(ctx,
     'strand': 'Align against reference sequences in forward ("plus") '
               'or both directions ("both").',
     'maxaccepts': 'Maximum number of hits to keep for each query. Set to '
-                  '"all" to keep all hits > perc_identity similarity.',
+                  '"all" to keep all hits > perc_identity similarity. Note '
+                  'that if strand=both, maxaccepts will keep N hits for each '
+                  'direction (if searches in the opposite direction yield '
+                  'results that exceed the minimum perc_identity). In those '
+                  'cases use maxhits to control the total number of hits '
+                  'returned. This option works in pair with maxrejects. '
+                  'The search process sorts target sequences by decreasing '
+                  'number of k-mers they have in common with the query '
+                  'sequence, using that information as a proxy for sequence '
+                  'similarity. After pairwise alignments, if the first target '
+                  'sequence passes the acceptation criteria, it is accepted '
+                  'as best hit and the search process stops for that query. '
+                  'If maxaccepts is set to a higher value, more hits are '
+                  'accepted. If --maxaccepts and --maxrejects are both set to '
```

Comment:
```suggestion
                  'accepted. If maxaccepts and maxrejects are both set to '
```

I think leaving the `--` in there might be more confusing than beneficial.